src/qt/rpcconsole.cpp

```diff
@@ -634,6 +651,7 @@ void RPCConsole::setClientModel(ClientModel *model, int bestblock_height, int64_
         ui->peerWidget->setColumnWidth(PeerTableModel::Subversion, SUBVERSION_COLUMN_WIDTH);
         ui->peerWidget->setColumnWidth(PeerTableModel::Ping, PING_COLUMN_WIDTH);
         ui->peerWidget->horizontalHeader()->setStretchLastSection(true);
+        ui->peerWidget->setItemDelegateForColumn(PeerTableModel::NetNodeId, new PeerIdViewDelegate(this));
```

Comment:
I don't have a strong opinion on this. I think with and without the `PeerIdViewDelegate` both look fine:

| With Delegate     | Without Delegate |
| ---------------- | ------------------ |
| ![Screen Shot 2021-05-12 at 9 46 31 PM](https://user-images.githubusercontent.com/23396902/118067831-55f82680-b36f-11eb-8888-29e7c2f51c33.png) | ![Screen Shot 2021-05-12 at 10 09 38 PM](https://user-images.githubusercontent.com/23396902/118067841-5abcda80-b36f-11eb-933c-1064a74b1de0.png) |

Reply:
Thanks for bringing this patch to my attention @jarolrod and for this helpful comparison. I prefer with the added separation between the right-aligned peer number and the left-aligned address (which is why I originally proposed center-aligned to have the spacing as well and aligned under the header that is also center-aligned, but the version proposed in this PR seems good too).